src/Meilisearch/PaginatedSearchResult.cs

```diff
@@ -93,5 +95,9 @@ public class PaginatedSearchResult<T> : ISearchable<T>
         /// </summary>
         [JsonPropertyName("facetStats")]
         public IReadOnlyDictionary<string, FacetStat> FacetStats { get; }
+
+        /// <inheritdoc/>
```

Comment:
This is much better than writing the description twice, as is currently the case! Can you do it for every field?
If it's too annoying, I can open an issue about it 😊

Reply:
https://github.com/meilisearch/meilisearch-dotnet/pull/416/commits/dc6c9b6c8769b2e297a43fa91ab6bdc23119a8df